src/main/java/underdogs/devbie/recommendation/service/QuestionRecommendationService.java

```diff
@@ -0,0 +1,53 @@
+package underdogs.devbie.recommendation.service;
+
+import static underdogs.devbie.recommendation.domain.RecommendationType.*;
+
+import org.springframework.stereotype.Service;
+
+import lombok.AllArgsConstructor;
+import underdogs.devbie.exception.AlreadyExistException;
+import underdogs.devbie.exception.NotExistException;
+import underdogs.devbie.recommendation.domain.QuestionRecommendation;
+import underdogs.devbie.recommendation.domain.QuestionRecommendationRepository;
+import underdogs.devbie.recommendation.domain.RecommendationType;
+import underdogs.devbie.recommendation.dto.RecommendationResponse;
+
+@Service
+@AllArgsConstructor
+public class QuestionRecommendationService {
+
+    private QuestionRecommendationRepository questionRecommendations;
```

Comment:
```suggestion
    private final QuestionRecommendationRepository questionRecommendations;
```

final을 붙이고 Lombok에 AllArgsContructor 대신 RequiredArgsConstructor를 사용하는 것은 어떨까요?